crates/pop-cli/src/commands/up/parachain.rs

```diff
@@ -97,12 +125,19 @@ impl UpChainCommand {
 	}
 }
 
+#[derive(Clone, Debug, PartialEq)]
+enum ProxyConfig {
```

Comment:
```suggestion
enum Proxy {
```

I dont see that the `Config` suffix adds any value based on how simple the enum definition is.

Reply:
Can you not achieve the same with `type Proxy = Option<String>;`?

Reply:
Yes, the type `Proxy = Option<String>;` seems enough. Refactored in this commit https://github.com/r0gue-io/pop-cli/pull/427/commits/f18de68e93111b9f14bde797ec587825157f3696